### PR TITLE
#184651052 HoundCI Configuration

### PR DESCRIPTION
**Pull request by Pacific911**
### What does this PR do?
- Configures HoundCI 

### Description of Task to be completed?
- Configuring HoundCI

### Any background context you want to provide?
- n/a

#### What are the relevant pivotal tracker?
- 184651052

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -1,24 +1,11 @@
...
+  "extends": "airbnb-base",
   "env": {
     "node": true,
     "es6": true,
-    "mocha": true,
-    "jest": true
+    "mocha": true
   },
   "rules": {
-    "prettier/prettier": "error",
```
Comment (Kress2000) [inline .eslintrc.json:21]: i think this has to be an error or at least a warning, as it will be used by the pre-commit hook

File: .eslintrc.json [hunk 2]
```diff
@@ -1,24 +1,11 @@
...
-    "airbnb-base",
-    "airbnb-base",
-    "prettier",
-    "airbnb/hooks",
-    "plugin:mocha/recommended",
-    "plugin:jest/recommended",
-    "prettier",
-    "plugin:prettier/recommended"
-  ],
-  "plugins": ["node", "prettier", "eslint-plugin-node", "jest"],
```
Comment (Kress2000) [inline .eslintrc.json:13]: explain why you removed plugins.

Comment (Pacific911) [reply .eslintrc.json:13]: check on the other rules they gave us in pivot tracker there were no plugins 
and another thing those rules are not supported by hound

**Comment by munezerobagira** [comment]
Good work @Pacific911 , you did a great job, but we using the ES6 syntax and we are getting lot of complaints on other PRs, if possible you can take look and see if you can fix them.

**Comment by Pacific911** [comment]
@munezerobagira I have been making the research trying to see how I can fix this  but I have found that this is not the problem of hound its because I configured the houndCI  to the repository and it checking everywhere to each branch where there is the wrong written line of codes with eslint so which means the other branch of eslint that was merged it consists of the eslint version that doesn't support/match with the HoundCI configuration  so which means my branch has to be merged so that it can match with the right version of eslint I have set  . You can check out too

**Review by munezerobagira — CHANGES_REQUESTED** [review]
Good work but you will have to resolve the conflict

**Pull request merged**